buildpacks/cnb.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: cnb
+spec:
+  parameters:
+  - name: IMAGE
+    description: The image you wish to create. For example, "repo/example", or "example.com/repo/image".
+  - name: RUN_IMAGE
+    description: The run image buildpacks will use as the base for IMAGE.
+    default: packs/run
+  - name: USE_CRED_HELPERS
+    description: Use Docker credential helpers for Google's GCR, Amazon's ECR, or Microsoft's ACR.
+    default: 'true'
+  - name: CACHE
+    description: The name of the persistent app cache volume
+    default: app-cache
+
+  steps:
+  - name: prepare
+    image: packs/base
+    command: ["/lifecycle/knative-helper"]
+    volumeMounts:
+    - name: "${CACHE}"
+      mountPath: /cache
+    imagePullPolicy: Always
+  - name: detect
+    image: packs/samples
```

Comment:
How about parameterizing this value, thus allowing users to specify the buildpack (or rather "buildpack-builder") they want to use? This parameter would apply to line 41, of course

Reply:
We can't do this until `knative-build` 0.2 is released. The currently released version doesn't support parameterizing images.